doc/association-rules.rst

```diff
@@ -0,0 +1,79 @@
+Association Rules
+=================
+
+.. figure:: icons/association-rules.png
+
+Induction of association rules.
+
+Signals
+-------
+
+#### Inputs
+
+-  **Data**
+
+Data set
+
+#### Outputs
+
+-  **Matching Data**
+
+Data instances matching the criteria.
+
+Description
+-----------
+
+This widget runs the original `Agrawal's algorithm <http://cs-people.bu.edu/evimaria/cs565/advances.pdf>`__ for the induction of
```

Comment:
Ah, sorry, [I see](https://orange3-associate.readthedocs.io/en/latest/#fpgrowth.association_rules). It would be much better to mention FP-growth instead of Agrawal or thier ancient article. Should save us some embarrassment. :smiley: